lib/components/AccountSetup.js

```diff
@@ -2,8 +2,23 @@ import React from 'react';
 
 const AccountSetup = ({submit}) =>
 <section className='account-setup'>
-  <h1>Account Setup</h1>
-<button onClick={submit}>Next</button>
+    <form>
+      <table className='setup-table'>
+        <tbody>
+      <tr>
+        <td className='tdQuest'>How much money do you want to spend this month?</td>
+        <td><input className='tdInput'/></td>
+        <td><button className='table-btn'></button></td>
+      </tr>
+      <tr>
+        <td className='tdQuest'>How much money do you want to spend on rent?</td>
```

Comment:
Cool looks like a good start, will play around in here today and think about your questions.